bin/Amc/Repl/Command.hs

```diff
@@ -60,6 +60,9 @@ execCommand :: (MonadState ReplState m, MonadIO m) => Listener -> String -> Stri
 execCommand tid "quit"  _   = finish tid
 execCommand tid "q"     _   = finish tid
 
+execCommand _ "?"       _   = helpCommand
+execCommand _ "help"    _   = helpCommand
```

Comment:
Maybe also just `h`?

Reply:
Sure. I think `:?` looks better, but `:h` is more consistent.